src/libraries/Common/src/System/Security/Cryptography/CompositeMLDsaManaged.cs

```diff
@@ -0,0 +1,682 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+using System.IO;
+using System.Runtime.Versioning;
+using Internal.Cryptography;
+
+namespace System.Security.Cryptography
+{
+#if !SYSTEM_SECURITY_CRYPTOGRAPHY
+    // System.Security.Cryptography excludes browser at build time, but we need to rely on UnsupportedOSPlatform for Microsoft.Bcl.Cryptography.
+    [UnsupportedOSPlatform("browser")]
```

Comment:
Pattern elsewhere is to only have this on members, but since this is an internal type that is not meant to be used outside of `CompositeMLDsaImplementation`, I don't expect us to need to relax the restriction at method level.